schema.sql

```diff
@@ -0,0 +1,68 @@
+CREATE TABLE genre (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE author (
+     id SERIAL PRIMARY KEY,
+     first_name VARCHAR(255),
+     last_name VARCHAR(255)
+);
+
+CREATE TABLE source (
+     id SERIAL PRIMARY KEY,
+     name VARCHAR(255)
+);
+
+CREATE TABLE label (
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(255),
+    color VARCHAR(255)
+);
+
+CREATE TABLE item (
+       id SERIAL PRIMARY KEY,
+       genre_id INT,
+       author_id INT,
+       source_id INT,
+       label_id INT,
+       publish_date DATE,
+       archived BOOLEAN,
+       FOREIGN KEY (genre_id) REFERENCES genres(id),
+       FOREIGN KEY (author_id) REFERENCES authors(id),
+       FOREIGN KEY (source_id) REFERENCES sources(id),
+       FOREIGN KEY (label_id) REFERENCES labels(id),
+       CHECK (publish_date IS NOT NULL),
+       CHECK (publish_date <= CURRENT_DATE),
+       CHECK (archived IN (TRUE, FALSE))
+);
+
+CREATE TABLE book (
```

Comment:
# Hi @mosidrum ,

### You've done a great job! 

### I think schema related to book and label class is assigned to your coding partner @anita00001 (Please check [issue 9](https://github.com/anita00001/Catalog-of-my-things/issues/9))
You may want to remove the table book.